themes/fast-track-theme/lms/templates/courseware/info.html

```diff
@@ -172,7 +172,7 @@ <h2 class="title">${_("You are not enrolled yet")}</h2>
                     <span class="date">
                       last viewed @ ${last_viewed_item['modified'].strftime("%B %d, %Y")}
                     </span>
-                    <a href=${'/courses/' + last_viewed_item['course_id'] + '/jump_to/' + last_viewed_item['module_id']} class="btn-ft">
+                    <a href=${'/courses/{}/courseware'.format(last_viewed_item['course_id'])} class="btn-ft">
```

Comment:
Instead of being smart, just send to courseware which knows where you last left off.